### PR TITLE
#45 Fix Hadron-typeorm incorrect repository names for PascalCase class/mo…

### PR DESCRIPTION
**Pull request by DanielMarkiel**
…del names

### CODE REVIEW

File: packages/hadron-typeorm/src/__tests__/connectionHelper.ts [hunk 1]
```diff
@@ -62,6 +63,11 @@ describe('TypeORM connection helper', () => {
       assert(Container.take('teamRepository')),
     ));
 
+  it('should register UserStatus repository to container as userstatusRepository', () =>
```
Comment (piotr-s-brainhub) [inline packages/hadron-typeorm/src/__tests__/connectionHelper.ts:66]: IMO it should be named `userStatusRepository`

Comment (DanielMarkiel) [reply packages/hadron-typeorm/src/__tests__/connectionHelper.ts:66]: Why? According to the documentation it shouldn't.
```
Repository keys in the container derive from names of schemas/classes and are built this way: <schema/class name in lower case>Repository

Examples:

User = userRepository
SuperUser = superuserRepository
loremIpsumDolor = loremipsumdolorRepository
```

Comment (DanielMarkiel) [reply packages/hadron-typeorm/src/__tests__/connectionHelper.ts:66]: Do you prefer to leave the current solution and change documentation instead?

Comment (piotr-s-brainhub) [reply packages/hadron-typeorm/src/__tests__/connectionHelper.ts:66]: I'm OK with leaving as it is

**Review by piotr-s-brainhub — APPROVED** [review]

**Pull request merged**